web/themes/joinup/prototype/sass/components/_form-table.sass

```diff
@@ -36,13 +44,53 @@ thead th.form-table__header
 
 // Dependencies
 // --------------------------------------------------
-.form-table__row
-  &.draggable
-    .draggable__icon, .form-managed-file
-      display: table-cell
+.form-table
+  .form-table__row
+    .dropbutton-multiple
+      min-height: auto
+      margin: 0
+      padding: 0
+    .dropbutton
+      display: flex
+    .dropbutton-widget
+      position: static
+    .dropbutton-toggle
+      display: none
+      visibility: none
+    .dropbutton-action
+      position: static
```

Comment:
My job was to style action links in the `form_table`. Unfortunately these links have dropdown markup which couldn't be changed, so I had to reset the styling and adjust links to requirements. In order to do so, first of all, to avoid overwriting of the already overwritten styles, I added a dependency to the existing styles for dropdown. These styles have been used in case of actions buttons in `form__actions` element which is completely different. After that I have added second dependency of the dropdown, but this time for `form__table`.

In the meantime, working on the `form__table`, I realised that this component is used in many places, some of the styling are duplicated and there is a chance for regression, so I decided to make minor adjustments and unifications.